src/Tribe/Editor/Configuration.php

```diff
@@ -39,15 +83,14 @@ public function editor_config( $editor_config ) {
 	public function localize() {
 		/** @var Tribe__Events__Admin__Event_Meta_Box $events_meta_box */
 		$events_meta_box = tribe( 'tec.admin.event-meta-box' );
-		return array(
+		$data = [
 			'settings'      => tribe( 'events.editor.settings' )->get_options(),
 			'timezoneHTML'  => tribe_events_timezone_choice( Tribe__Events__Timezones::get_event_timezone_string() ),
-			'priceSettings' => array(
+			'priceSettings' => [
 				'defaultCurrencySymbol'   => tribe_get_option( 'defaultCurrencySymbol', '$' ),
 				'defaultCurrencyPosition' => (
 					tribe_get_option( 'reverseCurrencyPosition', false ) ? 'suffix' : 'prefix'
-				),
-			),
+				), ],
```

Comment:
* :no_entry_sign: Closing parenthesis of array declaration must be on a new line
* :no_entry_sign: Array closer not aligned correctly; expected 12 space(s) but found 19
* :no_entry_sign: Closing parenthesis of array declaration must be on a new line

<sub>hash: 367e36be5f63104edb8b30285f05cc9f</sub>